app.js

```diff
@@ -338,8 +337,21 @@ function findCompilers() {
         });
     }
 
+    function getTypeFromBin(exe) {
```

Comment:
Ohh. The comment on the function made it seem like somewhat the final version of the function.
Then no complains on my part. Great work 😀